casper/src/test/scala/coop/rchain/casper/blocks/proposer/BlockReceiverStateSpec.scala

```diff
@@ -0,0 +1,116 @@
+package coop.rchain.casper.blocks.proposer
```

Comment:
To execute unit tests on CI, their folder must be specified in GitHub workflow file.
https://github.com/rchain/rchain/blob/4227faf20f9f8a50f29caca8129bce2b2c50a750/.github/workflows/continuous-integration.yml#L119-L130

You can add block receiver test to batch2 folder for now.

Reply:
I don't understand why you add this test to blocks **proposer** folder. Better to just add it to batch2 existing folder.

The tests are split in folders to run them on CI in parallel but having only one file in a folder will not make overall execution faster.
- #2936

Reply:
I see, but BlockReceiver is not in _proposer_ sub-folder.